### PR TITLE
Clean code and remove whitespace in markup layered navigation filters

### LINKED ISSUE #30448: [Issue] Clean code and remove whitespace in markup layered navigation filters
This issue is automatically created based on existing pull request: magento/magento2#30333: Clean code and remove whitespace in markup layered navigation filters

---------

### Description (*)

This pr aim to clean whitespace in some inline elements inside layered filter items. Also update use $escaper instead $block in template
### Fixed Issues (if relevant)

1. Fixes magento/magento2#<issue_number>

### Manual testing scenarios (*)

1. ...
2. ...

### Questions or comments

Luma theme
Results before change
before_remove

Results after change
after_remove

Blank theme
Results before change
blank_category

Results after change
blank_category_fix

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### PR DESCRIPTION
**Pull request by mrtuvn**
### Description (*)

This pr aim to clean whitespace in some inline elements inside layered filter items. Also update use $escaper instead $block in template
### Fixed Issues (if relevant)

1. Fixes magento/magento2#<issue_number>

### Manual testing scenarios (*)

1. ...
2. ...

### Questions or comments

Luma theme
Results before change
before_remove

Results after change
after_remove

Blank theme
Results before change
blank_category

Results after change
blank_category_fix

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### Resolved issues:
1. [x] resolves magento/magento2#30448: Clean code and remove whitespace in markup layered navigation filters

### CODE REVIEW

**Comment by ihor-sviziev** [comment]
@magento run all tests

File: app/code/Magento/LayeredNavigation/view/frontend/templates/layer/filter.phtml [hunk 1]
```diff
@@ -7,7 +7,8 @@
 // phpcs:disable Magento2.Templates.ThisInTemplate.FoundThis
```
Comment (ihor-sviziev) [inline app/code/Magento/LayeredNavigation/view/frontend/templates/layer/filter.phtml]: Looks like this line could be safely removed

**Review by ihor-sviziev — COMMENTED** [review]
In general looks good. Have only one suggestion

**Comment by mrtuvn** [comment]
@magento run Static Tests

**Comment by mrtuvn** [comment]
@magento run all tests

**Review by ihor-sviziev — APPROVED** [review]
✔ Approved.

Failing tests looks not related to changes form this PR.

**Comment by magento-engcom-team** [comment]
Hi @ihor-sviziev, thank you for the review.
[ENGCOM-8309](https://jira.corp.magento.com/browse/ENGCOM-8309) has been created to process this Pull Request

**Comment by ihor-sviziev** [comment]
@magento run Functional Tests B2B, Functional Tests CE, Functional Tests EE

**Comment by mrtuvn** [comment]
@magento run Functional Tests B2B, Functional Tests CE, Functional Tests EE

**Comment by mrtuvn** [comment]
@magento run Database Compare, Integration Tests, Magento Health Index, Sample Data Tests CE, Sample Data Tests EE, Sample Data Tests B2B, Static Tests, Unit Tests, WebAPI Tests

**Comment by mrtuvn** [comment]
@magento run Semantic Version Checker

**Comment by ihor-sviziev** [comment]
@magento run Functional Tests CE, Functional Tests EE

**Review by ihor-sviziev — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @ihor-sviziev, thank you for the review.
[ENGCOM-8309](https://jira.corp.magento.com/browse/ENGCOM-8309) has been created to process this Pull Request

**Comment by engcom-Alfa** [comment]
@magento create issue

**Comment by engcom-Alfa** [comment]
:heavy_check_mark: QA Passed

_Luma:_

**Before:** :heavy_multiplication_x: 



**After:** :heavy_check_mark: 



The same results and on the Blank theme

**Pull request merged**